components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/gen/java/org/wso2/carbon/apimgt/rest/api/admin/v1/ApplicationsApi.java

```diff
@@ -0,0 +1,74 @@
+package org.wso2.carbon.apimgt.rest.api.admin.v1;
+
+import org.wso2.carbon.apimgt.rest.api.admin.v1.dto.ApplicationListDTO;
+import org.wso2.carbon.apimgt.rest.api.admin.v1.dto.ErrorDTO;
+import org.wso2.carbon.apimgt.rest.api.admin.v1.ApplicationsApiService;
+import org.wso2.carbon.apimgt.rest.api.admin.v1.impl.ApplicationsApiServiceImpl;
+import org.wso2.carbon.apimgt.api.APIManagementException;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+import javax.inject.Inject;
+
+import io.swagger.annotations.*;
+import java.io.InputStream;
+
+import org.apache.cxf.jaxrs.ext.MessageContext;
+import org.apache.cxf.jaxrs.ext.multipart.Attachment;
+import org.apache.cxf.jaxrs.ext.multipart.Multipart;
+
+import java.util.Map;
+import java.util.List;
+import javax.validation.constraints.*;
+@Path("/applications")
+
+@Api(description = "the applications API")
+@Consumes({ "application/json" })
+@Produces({ "application/json" })
+
+
+public class ApplicationsApi  {
+
+  @Context MessageContext securityContext;
+
+ApplicationsApiService delegate = new ApplicationsApiServiceImpl();
+
+
```

Comment:
shall we remove this extra line?

Reply:
i think this is an autogenerated file. In that case we don't modify the file after generation.